snap/snapcraft.yaml

```diff
@@ -67,7 +67,6 @@ plugs:
   # the service-specific directory under $SNAP_DATA/secrets.
   edgex-secretstore-token:
     interface: content
-    content: edgex-secretstore-token
```

Comment:
Please revert this so that the PR only updates the necessary files.

Reply:
Thanks, reverted.